AK/Error.h

```diff
@@ -38,6 +38,15 @@ class Error {
         return Error(syscall_name, rc);
     }
     [[nodiscard]] static Error from_string_view(StringView string_literal) { return Error(string_literal); }
+    template<OneOf<DeprecatedString, DeprecatedFlyString> T>
```

Comment:
Add `String`, `FlyString`?

Reply:
`String` isn't implicitly convertible to `StringView` AFAIK?

Reply:
The constructors are implicit:
https://github.com/SerenityOS/serenity/blob/5141c865879bc8de6ffd98d4ea6f3fbd59b6434b/AK/StringView.h#L51-L52

Reply:
:thinking: I see... sure, will do.